File: Phase1_Frontend Application/src/app/Models/project.ts
// export interface Project {

// }
export class Project {
  //id:number,title:string,des:string,domain:string,tech:string,mail:string
  constructor() {
    this.projectId=0;
    this.projectTitle="";
    this.description="";
    this.domain="";
    this.techTools="";
    this.contactMail="";
   }
    projectId: number;
    projectTitle: string;
    description: string;
    domain: string;
    techTools: string;
    contactMail: string;
  }
  // public class Project
  // {
  //     [Key]
  //     [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
  //     public int ProjectId { get; set; }

  //     public string ProjectTitle { get; set; }

  //     public string Description { get; set; }

  //     public string Domain { get; set; }

  //     public string TechTools { get; set;}

  //     public string ContactMail { get; set; }
  
  // export interface Project {
  //   projectId: number;
  //   projectTitle: string;
  //   description: string;
  //   domain: string;
  //   techTools: string;
  //   contactMail: string;
  // }
  

File: Phase2_Web_API/Phase1_Projects_Backend/Models/Project.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Phase1_Projects_Backend.Models
{
    public class Project
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ProjectId { get; set; }

        public string ProjectTitle { get; set; }

        public string Description { get; set; }

        public string Domain { get; set; }

        public string TechTools { get; set; } = "To be Updated";

        public string ContactMail { get; set; }
    }
}
